src/hints/wideMul128.ts

```diff
@@ -0,0 +1,55 @@
+import { z } from 'zod';
+import { HintName } from 'hints/hintName';
+import { cellRef, CellRef } from 'hints/hintParamsSchema';
+import { resOperand, ResOperand } from 'hints/hintParamsSchema';
+import { VirtualMachine } from 'vm/virtualMachine';
+import { Felt } from 'primitives/felt';
+export const wideMul128Parser = z.
+    object({
+        WideMul128: z.object({
+            lhs: resOperand,
+            rhs: resOperand,
+            high: cellRef,
+            low: cellRef
+        }),
+    })
+    .transform(({ WideMul128: { lhs, rhs, high, low } }) => ({
+        type: HintName.WideMul128,
+        lhs: lhs,
+        rhs: rhs,
+        high: high,
+        low: low
+    }));
+
+export const wideMul128 = (
```

Comment:
add a well-formatted JSDoc, take example on the [`AllocFelt252Dict` one](https://github.com/kkrt-labs/cairo-vm-ts/blob/f035f5c1dac8e03b09895479935ea04983ce952b/src/hints/dict/allocFelt252Dict.ts#L28-L42)